source/speech/speech.py

```diff
@@ -181,7 +181,7 @@ def processText(
 	text = RE_CONVERT_WHITESPACE.sub(" ", text)
 	if normalize:
 		text = unicodeNormalize(text)
-	return text.strip()
+	return text.rstrip()
```

Comment:
this has been by design for 16 years, I assume to remove leading spaces in some scenarios. I would suggest maybe stripping `unicodeNormalize(text)` only for safety.

```suggestion
		# keep leading space for normalization message
		return text.rstrip()
	return text.strip()
```